src/puppet-web/bridge.ts

```diff
@@ -22,6 +22,7 @@ const retryPromise  = require('retry-promise').default
 import { log }    from '../config'
 
 import PuppetWeb  from './puppet-web'
+import {MsgRawObj}  from '../message'
```

Comment:
Change to `{ MsgRawObj }` please.

Also as @JasLin said: `Message` should not be coupled with PuppetWeb, so we should not `import` anything from `Message` from `PuppetWeb*`.

However, we can accept it now because those decouple work can be fixed in the future version.